plugins/parsers/csv/parser.go

```diff
@@ -197,6 +198,15 @@ outer:
 				value = strings.Trim(value, " ")
 			}
 
+			// don't record fields where the value matches a skip value
+			if len(p.SkipValues) > 0 {
```

Comment:
Is checking the length necessary?

Reply:
You're right it's redundant, I'll take it out